agent/engine/docker_container_engine.go

```diff
@@ -848,3 +849,11 @@ func (dg *dockerGoClient) removeImage(imageName string) error {
 	}
 	return client.RemoveImage(imageName)
 }
+
+func (dg *dockerGoClient) LoadImage(opts docker.LoadImageOptions) error {
+	client, err := dg.dockerClient()
```

Comment:
Sure but a lot of functions here don't have a timeout and we are only using this for loading a tiny pause container tarball. 
Could you please explain the problems you foresee ?